packages/migrate/src/commands/DbCommand.ts

```diff
@@ -14,16 +14,20 @@ ${chalk.bold('Usage')}
 
   ${chalk.dim('$')} prisma db [command] [options]
 
+${chalk.bold('Flag')}
+
+--preview-feature   Run Preview Prisma commands
+
```

Comment:
Maybe only after commands, but before Examples? Feels less important than Commands.

Reply:
I'll do that in the diff PR

Reply:
https://github.com/prisma/prisma/pull/11440